src/Wizard.js

```diff
@@ -352,7 +352,12 @@ export default class Wizard extends Webform {
         this.addEventListener(pageButton, 'click', (event) => {
           this.emit('wizardNavigationClicked', this.pages[i]);
           event.preventDefault();
-          this.setPage(i);
+           if (this.checkValidity(this.submission.data, true)) {
```

Comment:
nit:   what does `true` argument represent?

setting a constant can make the code more self-documenting, e.g.
```
const doDeepValidityCheck = true;
if (this.checkValidity(this.submission.data, doDeepValidityCheck)) {
```

note:  I have no idea what `true` is so obviously you would replace `doDeepValidityCheck` var name with whatever it actually represents.